packages/talos_certifier_adapters/src/mock_certifier_service.rs

```diff
@@ -40,16 +40,24 @@ impl SystemService for MockCertifierService {
                             safepoint: Some(0),
                             conflicts: None,
                             duplicate_version: None,
+                            can_published_at: 0,
```

Comment:
can we club them under a struct `metrics` or something and keep it, so that it is easier to interpret what is part of metrics and what are they other key data?

Reply:
Yes but not before we merge, I dont want to update anything in cohort before we get rid of the debug branch. However I had the same idea to add struct for metrics. Lets do it later.